src/detect-mark.c

```diff
@@ -253,81 +253,65 @@ static int DetectMarkPacket(DetectEngineThreadCtx *det_ctx, Packet *p,
 /**
  * \test MarkTestParse01 is a test for a valid mark value
  *
- *  \retval 1 on succces
- *  \retval 0 on failure
  */
 static int MarkTestParse01 (void)
 {
     DetectMarkData *data;
 
     data = DetectMarkParse("1/1");
```

Comment:
in modern code we would also declare the variable here to make the tests more compare